cmd/qgb/query/cmd.go

```diff
@@ -227,3 +239,63 @@ func printConfirms(logger tmlog.Logger, confirmsMap map[string]string, valset *c
 		logger.Info(addr, "number", i)
 	}
 }
+
+func writeConfirmsToJSONFile(logger tmlog.Logger, confirmsMap map[string]string, valset *celestiatypes.Valset, outputFile string) error {
+	signers, missingSigners := getSignersAndMissingSigners(confirmsMap, valset)
+	logger.Info("writing confirms json file", "path", outputFile)
+
+	file, err := os.Create(outputFile)
+	if err != nil {
+		return errors.Wrap(err, "error creating file")
+	}
+	defer func(file *os.File) {
+		err := file.Close()
+		if err != nil {
+			fmt.Println(err)
```

Comment:
[not blocking]
maybe use the logger here instead of printing?

Reply:
[e75850a](https://github.com/celestiaorg/orchestrator-relayer/pull/338/commits/e75850a148f16112f5201c675a836f6e9d35e537)